src/index/script/Palette/ColorSwatch.coffee

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import _ from 'lodash'
+import JunkDrawer from "../../../_common/script/JunkDrawer.coffee"
+
+export default class ColorSwatch extends React.Component
+
+	constructor: (props) ->
+		super props
+	
+	render: =>
+		<form onSubmit={this.processSelected}>
+			<button type="submit" className={"colorSwatch" + if this.props.isSelected then " selected" else ""}
+				id={"swatch-" + this.props.code} style={this.computeStyle()}></button>
```

Comment:
I like this - making sure the 'id' is unique

Reply:
Thank you! It's not really necessary to have IDs for these elements ... but you never know, it could be handy at some point.